Modules/Sources/Networking/Model/Copiable/Models+Copiable.generated.swift

```diff
@@ -5,6 +5,7 @@ import Codegen
 import Foundation
 import WooFoundation
 import struct Alamofire.JSONEncoding
+import struct NetworkingCore.JetpackSite
```

Comment:
I cannot compile the test target unless I add this import, which seems to come from a missing `rake generate` somewhere else.

Reply:
I tried removing this line, building the app then ran `POSCouponTests`, both the app and tests ran. I've had frustrating times when it took a few tries rebuilding/cleaning though. If you get to build and run tests without this line, let's remove this.